api/swagger.yml

```diff
@@ -3310,3 +3311,19 @@ paths:
                 $ref: "#/components/schemas/StorageConfig"
         401:
           $ref: "#/components/responses/Unauthorized"
+
+  /config/emailer:
+    get:
+      tags:
+        - config
+      operationId: getEmailConfig
+      description: retrieve emailer config
+      responses:
+        200:
+          description: emailer configuration
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
Please specify an object type. Otherwise it will not be possible to use the API defined in this file to develop code.